packages/terra-date-picker/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+Fix issue where tooling was parsing the .scss import inside of our .module.scss file as a CSS modules file
```

Comment:
Are there screenshot tests for the date picker that should have broke?

Reply:
No, our webpack config parsed these files as intended. If you view [the page on terra-ui](https://engineering.cerner.com/terra-ui/#/components/terra-date-time-picker/date-time-picker/date-time-picker), it renders the datepicker correctly. 

It only appeared to come up with the way webpacker loaders parsed this file and it's import. With this change, our webpack config still parses it correctly, we've just made it more explicit the third-party styles should be global for other loader configs.